johnsonandjohnson/Bodiless-JS
# Fix/migration tool typeerror tag primaryattributekey tolowercas

## Changes

Fixing 'TypeError: tag[primaryAttributeKey].toLowerCase is not a function' on build step

## Test Instructions


1. Setup migration configuration
```
{
  "url": "https://lepetitmarseillais.be/fr/soins-du-visage/lingettes-micellaires-demaquillantes-leau-florale-de-rose",
  "crawler": {
    "maxDepth": 1,
    "maxConcurrency": 1,
    "ignoreRobotsTxt": true
  },
  "transformers": []
}

```

2. `npm run migrate`

3. `npm run build`

Expected: no build error.

## Related Issues

Fixes #215
